mantle/platform/api/aws/network.go

```diff
@@ -208,13 +209,22 @@ func (a *API) createRouteTable(vpcId string) (string, error) {
 		return "", fmt.Errorf("creating remote route: %v", err)
 	}
 
+	_, err = a.ec2.CreateRoute(&ec2.CreateRouteInput{
+        DestinationIpv6CidrBlock: aws.String("::/0"),
+		GatewayId:            aws.String(igw),
+		RouteTableId:         rt.RouteTable.RouteTableId,
+	})
+	if err != nil {
+		return "", fmt.Errorf("creating remote route: %v", err)
+	}
+
```

Comment:
Looks like this ended up in this commit by mistake.

Reply:
fixed!